app/src/main/java/io/homeassistant/companion/android/sensors/LightSensorManager.kt

```diff
@@ -0,0 +1,84 @@
+package io.homeassistant.companion.android.sensors
+
+import android.content.Context
+import android.hardware.Sensor
+import android.hardware.SensorEvent
+import android.hardware.SensorEventListener
+import android.hardware.SensorManager.SENSOR_DELAY_NORMAL
+import androidx.appcompat.app.AppCompatActivity
+import io.homeassistant.companion.android.domain.integration.SensorRegistration
+import kotlin.math.roundToInt
+
+class LightSensorManager : SensorManager, SensorEventListener, AppCompatActivity() {

```

Comment:
I don't think we want this to extend an Activity with a sensor.  An activity should only be used to display things.

Reply:
Yea now that we don't need `onPause` I will remove this